### ISSUE TITLE
WTFcsv remove drag handle code for accessibility

### ISSUE THREAD

**Issue by kanarinka** [c4fcm/DataBasic#95]
I also noticed references to drag handles, etc. What ways of manipulating the data are available to sighted mouse users? Are there ways of duplicating this functionality via the keyboard?  If its just manipulating pictures / charts / graphs, then making it keyboard accessible makes no sense since screen reader users cannot use graphics.

**Comment by jayvachon** [comment]
this was an old class name from when cards could be moved around the page, so I just renamed the class
